packages/core/jsr.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@zod/zod",
```

Comment:
shouldn't this be `@zod/core` or something?

Reply:
Updated. Totally forgot about JSR.